.github/workflows/show-commit.yaml

```diff
@@ -0,0 +1,16 @@
+name: Show commit
```

Comment:
It's not the same structure as a workflow. See composite example here https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file

Reply:
Oh thx! I've been taking this from a different example that was structured as a workflow. Let's see if it works after [this force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/a8fbae2e719dbd7d48171892aa770a2ba28dbc4c..21e25874ccdd085f153549c6b353e985107098fa).